src/main/java/frc/robot/generated/TunerConstants.java

```diff
@@ -80,7 +78,7 @@ public class TunerConstants {
 
     // CAN bus that the devices are located on;
     // All swerve devices must share the same CAN bus
-    public static final CANBus kCANBus = new CANBus("can_s1", "./logs/example.hoot");
+    public static final CANBus kCANBus = new CANBus("can_s0", "./logs/example.hoot");
```

Comment:
lets get used to using the new syscore canbus api
```suggestion
    public static final CANBus kCANBus = CANBus.systemCore(0, "./logs/example.hoot");
```